package.json

```diff
@@ -4,27 +4,32 @@
   "private": true,
   "scripts": {
     "dev": "next dev --turbopack",
-    "build": "next build",
+    "build": "prisma generate && next build",
```

Comment:
can we add another script more migrating the db? We're going to need to do that from time to time?